odata-to-abstract-sql.ometajs

```diff
@@ -16,8 +16,8 @@ Query.prototype.merge = function(otherQuery) {
 	this.where = this.where.concat(otherQuery.where);
 	this.extras = this.extras.concat(otherQuery.extras);
 };
-Query.prototype.fromResource = function(resource, args) {
-	if (resource.definition) {
+Query.prototype.fromResource = function(resource, args, bypassDefinition) {
+	if (bypassDefinition !== true && resource.definition) {
```

Comment:
What is the reason you explicitly test for `true`?

Reply:
Because I want to make sure that the value has been explicitly set to true, not just to a truth value, since bypassing the definition is an action that should be very rare and done explicitly (because bypassing the definition of a table is a special case and bypasses any translations/filters/etc)